### ISSUE TITLE
ocropy vs tesseract

### ISSUE THREAD

**Issue by ChillarAnand** [tmbdev/ocropy#235]
I thought tesseract gives better accuracy than ocropus as per these[1] [2] [3] resources and they seem to be outdated. I have training a telugu model with ocropy is giving excellent results.

I would like to know how different is tesseract(V4 seems to use LSTM) from ocropy and where does each excel? If ocropy authors or someone who have worked with both OCRs, provide some insights, that would be great. 

[1]: https://stackoverflow.com/a/10040372/2698552
[2]: https://www.reddit.com/r/MachineLearning/comments/3dcr6y/is_ocropus_really_worse_than_tesseract_as_people/
[3]: https://github.com/tesseract-ocr/tesseract/issues/630#issuecomment-269792095

**Comment by amitdo** [comment]
Tesseract 4.00 LSTM based OCR engine is basically a C++ reimplementation of the LSTM code in ocropy 0.7. Tesseract 4.00 added 2D-LSTM and uses language model (word lists).

Binarization and layout analysis are different between the two.

**Comment by zuphilip** [comment]
The trained models in Tesseract is AFAIK mostly on synthetical data and currently not clear how difficult it is (or is it still possible?) to train a new model by yourself. Another difference is that Tesseract uses a language dictionary which tries to correct mispelled/misrecognized words. Such a postocr step could be still performed after the recognition by ocropy, e.g. with PoCoTo.

**Comment by ChillarAnand** [comment]
There is an option to train tesseract but it is a pain to train tesseract. 

@amitdo @zuphilip  Thanks for the clarification.

**Comment by amitdo** [comment]
https://github.com/tesseract-ocr/tesseract/search?q=%22lstm.py%22

>// C++ Implementation of the LSTM class from lstm.py.

>// C++ Implementation of the Softmax (output) class from lstm.py.

>// C++ Implementation of the Reversed class from lstm.py.

**Comment by amitdo** [comment]
(April 2016)
https://github.com/tesseract-ocr/docs/blob/master/das_tutorial2016/6ModernizationEfforts.pdf

### LSTM Integration

● LSTM code based on OCROpus Python implementation.
● Expanded capabilities including 2-D, variable input sizes.
● Fully integrated with Tesseract at the group-of-similar-words level.
● Visualization with existing Viewer API.
● Training code included (unlike cube).
● Parallelized with openMP.
● Coming in next release of Tesseract.
